obspy/scripts/runtests.py

```diff
@@ -10,7 +10,7 @@
     --help : Print help message an exit.
     --no-report, --report : Do not ask and automatically (not) submit report
             to ObsPys' report server.
-    --cov : Calculate and display test coverage.
+    --cov obspy: Calculate and display test coverage.
```

Comment:
Actually just using `obspy-runtests --cov` is working fine for me (but does not display missing files).

Reply:
I think `--cov` will just report the coverage on files that were imported. `--cov obspy` will report all files in ObsPy, even those not imported, as 0% coverage.  You can see the difference if you run 

```bash
obspy-runtest io/sac --cov
```

vs

```bash
obspy-runtests io/sac --cov obspy
```